tests/unit_tests/test_data_generators_generators.py

```diff
@@ -0,0 +1,39 @@
+import unittest
+from datetime import datetime
+
+from data_generators.generators import percent_true, random_first_name, random_datetime
+
+
+class TestRandomizerFunctions(unittest.TestCase):
+
+    def test_percent_true_input(self):
+        self.assertFalse(percent_true(-100))
+        self.assertTrue(percent_true(1000))
+
+        with self.assertRaises(TypeError):
+            percent_true('String')
+        with self.assertRaises(TypeError):
+            percent_true(55.55)
```

Comment:
Are you sure this should be a failure? What if we changed the function signature to `percent_true(Float)`? Does it break the app?

Reply:
Definitely a signature-function mismatch where the IDE says error but the code itself works fine with floats as well as the specified int. Is best option to flip the test and fix the function to accept float at a later date?

Reply:
If a change to the code only breaks a test, I would say the problem is in the test.

Reply:
Agreed, upon reflection. Will be refactored in the revised commit